src/compiler/diagnosticMessages.json

```diff
@@ -3607,6 +3607,14 @@
         "category": "Error",
         "code": 17013
     },
+    "JSX fragment has no corresponding closing tag.": {
+        "category": "Error",
+        "code": 17014
+    },
+    "Expected corresponding JSX fragment closing tag.": {

```

Comment:
Expected corresponding closing tag for JSX fragment.

Reply:
Updated.